test/integration/run_envoy_test.sh

```diff
@@ -52,6 +52,6 @@ expect_ok -c "${TEST_SRCDIR}/envoy/test/config/integration/empty.yaml" --mode va
 
 start_test "Launching envoy with empty config."
 run_in_background_saving_pid "${ENVOY_BIN}" -c "${TEST_SRCDIR}/envoy/test/config/integration/empty.yaml" --use-dynamic-base-id
-sleep 3
+sleep 120
```

Comment:
Is this change intentional?

Reply:
Yes, otherwise the test fails in coverage. It takes over 1 minute in my tests to load coverage instrumented dynamically linked binary.